packages/strapi-design-system/src/v2/Breadcrumbs/CrumbSimpleMenu.js

```diff
@@ -22,9 +22,11 @@ export const CrumbSimpleMenu = ({ children, ...props }) => (
   </SimpleMenu>
 );
 
+const menuItemType = PropTypes.shape({ type: PropTypes.oneOf([MenuItem]) });
+
 CrumbSimpleMenu.displayName = 'CrumbSimpleMenu';
 
 CrumbSimpleMenu.propTypes = {
   'aria-label': PropTypes.string.isRequired,
-  children: PropTypes.arrayOf(SimpleMenu),
+  children: PropTypes.oneOfType([PropTypes.arrayOf(menuItemType), menuItemType]).isRequired,
```

Comment:
Weird issue here:
- In storybook it works well
- When using it in Strapi we are still having an issue saying "_wrong children passed to CrumbSimpleMenu + SimpleMenu_", no matter which MenuItem version used (v1 or v2)
- I tried to allow both version but since I have to rename at least one MenuItem component with `as` it still considers it as a different component

Any idea is welcomed - error will keep appear in Strapi without fixing it.
Other solution:
Accepting any node as for Breadcrumbs children but 🤷🏻‍♀️

Reply:
@HichamELBSI would love your opinion on this one if you have some time, maybe you have some knowledge about why in Strapi this component would act like we are not passing the right component version 
is something happening during the build that would make it confusing for this component to have 2 MenuItem components? (which I would find weird but that's my only theory so far)

Reply:
We are having the same issue on Cloud. I personally think that we shouldn't restrict the children proptype and keep the default react children. If we need a children with specific properties, we can do some checks before cloning the children.

Reply:
thank you for your answer @HichamELBSI!
to make sure I understand "we shouldn't restrict the children proptype": 
`SimpleMenu` should restrict its children to be `MenuItem` components only so we would have `children: PropTypes.node` ?